modules/labs-react/radio/lib/RadioInput.tsx

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
+import {
+  styled,
+  StyledType,
+  focusRing,
+  mouseFocusBehavior,
+  createSubcomponent,
+  createElemPropsHook,
+} from '@workday/canvas-kit-react/common';
+import {
+  colors,
+  inputColors,
+  spaceNumbers,
+  borderRadius,
+  space,
+} from '@workday/canvas-kit-react/tokens';
+import {useRadioModel} from './hooks/useRadioModel';
+import {RadioLabelProps, RadioLabelContext} from './RadioLabel';
+import {Box, Flex} from '@workday/canvas-kit-react/layout';
+
+const radioWidth = 18;
+const rippleRadius = (spaceNumbers.l - radioWidth) / 2;
+const radioHeight = 18;
+
+const StyledRadioInput = styled(Box.as('input'))<RadioLabelProps & StyledType>(
+  {
+    '&:focus, &:active': {
+      outline: 'transparent',
+    },
+  },
+  ({
+    disabled,
+    variant,
+    theme: {
+      canvas: {
+        palette: {
+          primary: themePrimary,
+          common: {focusOutline: themeFocusOutline},
+        },
+      },
+    },
+  }) => ({
+    cursor: disabled ? undefined : 'pointer',
+    opacity: disabled && variant === 'inverse' ? '.4' : '1',
+    height: '18px',
+    width: '18px',
+
+    // Circle element styles the radio as checked or unchecked
+    ':after': {
+      content: "''",
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+      backgroundColor: disabled ? inputColors.disabled.background : colors.frenchVanilla100,
+      borderRadius: '999px',
+      boxSizing: 'border-box',
+      border: `1px solid`,
+      borderColor: disabled
+        ? inputColors.disabled.border
+        : variant === 'inverse'
+        ? colors.soap300
+        : inputColors.border,
+      height: '18px',
+
+      width: '18px',
+      justifyContent: 'center',
+      paddingY: '0px',
+      paddingX: '2px',
+      pointerEvents: 'none',
+      position: 'absolute',
+      transition: 'border 200ms ease, background 200ms',
+      opacity: disabled && variant === 'inverse' ? '.4' : '1',
+    },
+
+    '&:checked:after': {
+      backgroundColor: variant === 'inverse' ? themePrimary.main : colors.frenchVanilla100,
+      border: `5px solid`,
```

Comment:
This is what gives the radio `selected` state it's color. The background is the small circle and the border is the larger circle for the selected state.